docs/DeveloperGuide.md

```diff
@@ -1225,22 +1226,143 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### 7.2 Adding a patient
 
-### 7.2 Deleting a patient
+1. Add a patient while all patients are being shown
+
+   1. Prerequisites: List all patients using the `list` command. Multiple patients in the list.
+
+   1. Test case: `add id/ 54321 n/ John Doe p/ John f/ Curry chicken c/ Stable h/ Singing karaoke t/ amnesia`<br>
+      Expected: New patient is added to the list. Status message shows details of the new patient.
+
+      1. Other incorrect add commands to try: `add`, `add n/`<br>
+         Expected: Error message displayed.
+
+### 7.3 Deleting a patient
 
 1. Deleting a patient while all patients are being shown
 
-   1. Prerequisites: List all patients using the `list` command. Multiple patients in the list.
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. 
 
    1. Test case: `delete 0`<br>
-      Expected: No patient is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No patient is deleted. Error details shown in the status message.
+
+   1. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size)<br>
+      Expected: Error message displayed.
+
+### 7.4 Listing all patients
+
+1. Listing all patients
+
+   1. Prerequisites: Patient List should not be empty.
+
+   1. Test case: `list`<br>
+      Expected: All patients are shown in the list.
+
+### 7.5 Editing a patient
+
+1. Editing a patient while all patients are being shown
+
+   1. Prerequisites: List all patients using the `list` command. Patient List should not be empty.
+
+   1. Test case: `edit 1 p/Alex f/Fried rice`<br>
+      Expected: The preferred name of the first patient is changed to Alex.
```

Comment:
Would be better if the `Expected:` section could add more explanation like `food preference of first patient is changed to Fried rice`.